bindata/scripts/add-rdma-service.sh

```diff
@@ -0,0 +1,39 @@
+echo '# SR-iOV Operator RDMA service
+
+RETRIES=100
+WAIT_TIME=1
+
+new_rdma_mode=
+
+rdma_mode=$(rdma system | cut -c7- | grep "[a-z]")
+
+if [ "$rdma_mode" = "$new_rdma_mode" ]; then
+  exit 0
+fi
+
+for _ in $(seq 0 $RETRIES); do
+  rdma system set netns "$new_rdma_mode"
+  ret_val=$?
+  if [ "$ret_val" -eq 0 ]; then
+    exit 0
+  fi
+  sleep $WAIT_TIME
+done' > /etc/init.d/sriov-network-rdma.sh
+
+rdma_mode=$(rdma system | cut -c7- | grep "[a-z]")
+
+sed -i 's/new_rdma_mode=.*/new_rdma_mode='$rdma_mode'/g' /etc/init.d/sriov-network-rdma.sh
+
+cat > /usr/lib/systemd/system/sriov-network-operator-rdma.service <<EOF
```

Comment:
Do we have to use a systemd service to change the rdma mode? And in Openshift, we prefer to use MCO to manage all the systemd services.

Reply:
@pliurh yes, changing rdma mode fails when there are containers running
For openshift MCO it can be in a follow-up PR